### ISSUE TITLE
Some issues about the use of KL Loss

### ISSUE THREAD

**Issue by CSer-Tang-hao** [chxy95/Deep-Mutual-Learning#2]
Hi, I am interested in this implement of "Deep Mutual Learning", but I have some uncertainty. In the code of trainer.py, you use "F.log_softmax(outputs[i]" for model1, but using "F.softmax(Variable(outputs[j])" for model2. Why?
 Looking forward to hearing from you!

**Comment by chxy95** [comment]
You can read the explanation of the function '**torch.nn.KLDivLoss()**' in the documents of Pytorch.
its' formulated as **D( p || q ) = kl_div ( log (q),  p )**.

**Comment by CSer-Tang-hao** [comment]
> You can read the explanation of the function '**torch.nn.KLDivLoss()**' in the documents of Pytorch.
> its' formulated as **D( p || q ) = kl_div ( log (q), p )**.

Thanks! I have noticed it
